### ISSUE TITLE
Would like some help

### ISSUE THREAD

**Issue by ghost** [aonurdemir/Mephisto#1]
Dear Mr.  DEMIR,

I've been trying to use this awesome software and I also read your thesis through the link in this repo but I don't have enough knowledge so I would appreciate it if you could let me know how to get this working.
(I'm not sure if this information helps but I can use linux mint 20.3 or windows 10.)
Thank you for your help in advance.

Best Regards,
Kyosuke

**Comment by magnetophon** [comment]
Hi @Kyosuke-Satake,

I think you need to be a bit more specific:  what doesn't work; which errors do you see?

**Comment by Kyosuke-Satake** [comment]
Hi @magnetophon,

Thank you so much for your reply and tips.
Actually, I'm completely new to java and I would like to know how to compile this java source code.
(Again, not sure if this helps but I downloaded eclipse IDE coz I saw the word "eclipse" in the source file.)

**Comment by magnetophon** [comment]
TBH, I have never compiled this and don't know java either.
Don't take it the wrong way, but it seems like you could benefit from reading this:
http://www.catb.org/~esr/faqs/smart-questions.html
tldr: Give it a shot, and when you get stuck, come back and explain where you got stuck.

**Comment by Kyosuke-Satake** [comment]
Thank you for your reply.
I have been doing my search on the internet but I still can't get close to successfully compile this and even if I could get there I probably have to figure out how to "use" this transpiler next.
So I think it would be easier for me to write code in faust itself than to use this transpiler.(at least for now)
Anyways, thank you for trying to help me and your tips.

**Comment by aonurdemir** [comment]
Hi mate 👋 I have added an extended documentation for the project. I hope it helps. 

Have a great day.

**Comment by ghost** [comment]
Hi,
Thank you so much for the documentation.
It was clear and easy to understand even for someone like me.

Have a great day.
